packages/providers/src/lib/push/one-signal/one-signal.provider.ts

```diff
@@ -38,8 +46,17 @@ export class OneSignalPushProvider
   ): Promise<ISendMessageSuccessResponse> {
     const { sound, badge, ...overrides } = options.overrides ?? {};
 
+    const targetSegment =
+      this.apiVersion === 'externalId'
+        ? {
+            include_aliases: {
+              external_id: options.target,
+            },
+          }
```

Comment:
```suggestion
        ? {
            include_aliases: {
              external_id: options.target,
            },
            target_channel: 'push',
          }
```

I just noticed in the docs that we would need to include the target_channel for the new API endpoint:

<img width="1149" alt="image" src="https://github.com/user-attachments/assets/72a16edc-0f91-4a82-82cb-0b4651ff7244">

Reply:
Yes this si good addition target Chandlera pu#h